loadimpactcli/userscenario_commands.py

```diff
@@ -59,7 +59,9 @@ def list_scenarios(project_id):
 @click.argument('script_file', type=click.File('r'))
 @click.argument('name')
 @click.option('--project_id', default=DEFAULT_PROJECT, envvar='DEFAULT_PROJECT', help='Id of the project the scenario should be in.')
-def create_scenario(script_file, name, project_id):
+@click.option('--datastore_file', type=click.File('r'), multiple=True, help='A CSV file to be used as a new data store for the user scenario. The name of the file is used as a name for the data store. Multiple files can be provided by repeating the option.')
```

Comment:
I think we should add something here about the delimiter being a double, separator a comma and that we will read from line 1. Since we've hardcoded that for now.